CMakeLists.txt

```diff
@@ -1055,7 +1055,6 @@ if(APPLE)
     target_link_libraries(${bun} PRIVATE "${ICU4C_DIR}/lib/libicudata.a")
     target_link_libraries(${bun} PRIVATE "${ICU4C_DIR}/lib/libicui18n.a")
     target_link_libraries(${bun} PRIVATE "${ICU4C_DIR}/lib/libicuuc.a")
-    include_directories(${ICU4C_DIR}/include)

```

Comment:
```suggestion
    include(${ICU4C_DIR}/include/iconv.h)
```

Reply:
On second thought maybe the code suggestion of the specific header to include... we can do this for other areas using include_directories too.

Reply:
Unfortunately, I don't think CMake allows you to include just a single header file? I'm pretty sure the CMake `include` command is used to include CMake code from a different file, not add a header file to the build. Making the change and then running `bun run build` gives an error like:
```
CMake Error at /opt/local/include/iconv.h:1:
  Parse error.  Expected a command name, got unquoted argument with text
  "/*".
Call Stack (most recent call first):
  CMakeLists.txt:1089 (include)
```
since a header file is obviously not valid CMake code. Based on some googling, I think that the only thing we can do is control is the order in which directories get included (by actually reordering the `include_directories` or using BEFORE).

Reply:
we shouldnt do this change. `include` is fine